Mathlib/RingTheory/MatrixAlgebra.lean

```diff
@@ -146,5 +154,59 @@ theorem matrixEquivTensor_apply_stdBasisMatrix (i j : n) (x : A) :
 
 @[simp]
 theorem matrixEquivTensor_apply_symm (a : A) (M : Matrix n n R) :
-    (matrixEquivTensor R A n).symm (a ⊗ₜ M) = M.map fun x => a * algebraMap R A x :=
+    (matrixEquivTensor R A n).symm (a ⊗ₜ M) = a • M.map (algebraMap R A) :=
   rfl
+
+namespace Matrix
+open scoped Kronecker
+
+variable (m) (B) [Fintype m] [DecidableEq m]
+
+/--
+`Matrix.kroneckerTMul` as an algebra equivalence, when the two arguments are tensored.
+
+TODO:
+* Generalize this to rectangular matrices
```

Comment:
What does this mean exactly? Rectangular matrices don't form a ring so you can't hope for an AlgEquiv or even RingHom.